src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -15,7 +15,8 @@
   <PropertyGroup>
     <PrepareForRunDependsOn Condition="'$(IsTestProject)'=='true'">$(PrepareForRunDependsOn);CopyTestToTestDirectory;CreateTestNuGetPackage;ArchiveTestBuild</PrepareForRunDependsOn>
     <TestArchitecture Condition="'$(TestArchitecture)' == ''">x64</TestArchitecture>
-    <TestNugetRuntimeId Condition="'$(TestNugetRuntimeId)' == ''">win7-$(TestArchitecture)</TestNugetRuntimeId>
+    <TestNugetRuntimeId Condition="'$(TestNugetRuntimeId)' == '' and '$(TestArchitecture)'!='arm64'">win7-$(TestArchitecture)</TestNugetRuntimeId>
+    <TestNugetRuntimeId Condition="'$(TestNugetRuntimeId)' == '' and '$(TestArchitecture)'=='arm64'">win10-$(TestArchitecture)</TestNugetRuntimeId>
```

Comment:
I actually ended up doing that anyways. You're right, this isn't necessary. I'll revert that bit.